src/helm/benchmark/static/schema.yaml

```diff
@@ -2954,540 +2984,560 @@ run_groups:
       language: synthetic
 
 ## CLEVA (Chinese) Scenarios
-# TODO: Fill in description and taxonomy
-  - name: cleva_bias
-    display_name: CLEVA (Chinese) bias
-    description: CLEVA (Chinese) bias
+# Applications
+  - name: cleva_closed_book_question_answering
+    display_name: CLEVA (Chinese) closed book question answering
+    description: Closed-book question answering task comprises three subtasks. One is for the medical domain, another for open-domain, and the last measures if a model generates truthful answers.
     metric_groups:
       - accuracy
-      - calibration
+      - efficiency
       - general_information
     environment:
       main_name: exact_match
       main_split: test
     taxonomy:
-      task: "?"
-      what: "?"
-      who: "?"
-      when: "?"
+      task: question answering
+      what: medical, open-domain, or truthful qa
+      who: n/a
+      when: n/a
       language: Chinese
 
-  - name: cleva_classical_chinese_understanding
-    display_name: CLEVA (Chinese) classical Chinese understanding
-    description: CLEVA (Chinese) classical Chinese understanding
+  - name: cleva_summarization
+    display_name: CLEVA (Chinese) summarization
+    description: "Contains one subtask so far: summarize a dialogue between a customer representative and a customer."
     metric_groups:
-      - accuracy
-      - calibration
+      - cleva_summarization_metrics
       - general_information
+      - efficiency
     environment:
-      main_name: exact_match
+      main_name: chinese_rouge_2
       main_split: test
     taxonomy:
-      task: "?"
-      what: "?"
-      who: "?"
-      when: "?"
+      task: summarization
+      what: e-commerce dialogues
+      who: customers and representatives
+      when: n/a
```

Comment:
If you know the time period (i.e. years) in which the data is collected, fill it in under "when". Otherwise, "n/a" is fine. Same for the other scenarios.

Reply:
Thanks for your suggestion. We marked the data collection time for most scenarios.